Detector/DetCommon/src/SimpleCylinder_geo.cpp

```diff
@@ -6,20 +6,26 @@ namespace det {
   @author Clement Helsens
 **/
 static DD4hep::Geometry::Ref_t
-createSimpleCylinder(DD4hep::Geometry::LCDD& lcdd, xml_h e, DD4hep::Geometry::SensitiveDetector) {
+createSimpleCylinder(DD4hep::Geometry::LCDD& lcdd, xml_h e, DD4hep::Geometry::SensitiveDetector sensDet) {
   xml_det_t x_det = e;
   std::string name = x_det.nameStr();
   DD4hep::Geometry::DetElement cylinderDet(name, x_det.id());
-
+  
```

Comment:
This file should be checked with checkformat to remove those trailing whitespaces.